src/reactionsystem.jl

```diff
@@ -1414,6 +1425,20 @@ function Base.convert(::Type{<:SDESystem}, rs::ReactionSystem;
               kwargs...)
 end
 
+# Extracts any noise scaling parameters from a reaction system.
+function get_noise_scaling(rs::ReactionSystem)
```

Comment:
Generally MTK get functions only return what is in the top level system by convention, i.e. `get_ps(rs)` vs. `parameters(rs)`. Maybe make two versions `get_noise_scalings` and `noise_scalings` that are analogous to `get_ps` and `parameters`. Also, to be consistent with other MTK functions I'd just return the vector from `filter` and not have this variable return type.

Reply:
Oh never mind. I missed that this is internal only right? So don't worry about this comment. Since it will only be called within `SDEProblem` this shouldn't be an issue.